js/main_nav.js

```diff
@@ -17,6 +25,16 @@ navBurguer.addEventListener("click", function () {
   }
 });
 
+// This function recalculates the values of logo's height and the space between logo and the very top of the viewport when a user makes resize of the window
```

Comment:
please reference the styleguide to document your code
https://github.com/airbnb/javascript#comments